virtme/commands/run.py

```diff
@@ -1282,25 +1282,27 @@ def do_it() -> int:
     if args.graphics is None and not args.script_sh and not args.script_exec:
         qemuargs.extend(["-echr", "1"])
 
-        if args.verbose:
-            # Check if we have permission to access the current stderr.
-            if not can_access_file("/proc/self/fd/2"):
-                print(
-                    "ERROR: not a valid pts, try to run vng inside tmux or screen",
-                    file=sys.stderr,
-                )
-                sys.exit(1)
-
-            # Redirect kernel messages to stderr, creating a separate console
+        # Redirect kernel errors to stderr, creating a separate console.
+        #
+        # If we don't have access to stderr via procfs (for example when
+        # running inside a container), print a warning and implicitly
+        # supress the kernel errors redirection.
```

Comment:
(just in case you have something else to modify)

```suggestion
        # suppress the kernel errors redirection.
```

Reply:
Fixed, thanks! :)

And I also slightly changed the warning to clarify that kernel errors are suppressed if we can't find a valid pts.